ci/prow/boskos/create_projects.sh

```diff
@@ -16,18 +16,29 @@
 
 set -e
 
-readonly FIRST=${1:?"First argument is the first number of the new project(s)."}
-readonly NUMBER=${2:?"Second argument is the number of new projects."}
-readonly BILLING_ACCOUNT=${3:?"Third argument must be the billing account."}
-readonly OUTPUT_FILE=${4:?"Fourth argument should be a file name all project names will be appended to in a resources.yaml format."}
+readonly NUMBER=${1:?"First argument is the number of new projects to create."}
+readonly BILLING_ACCOUNT=${2:?"Second argument must be the billing account."}
 
-for (( i=0; i<${NUMBER}; i++ )); do
-  PROJECT="knative-boskos-$(( i + ${FIRST} ))"
+readonly RESOURCE_FILE="resources.yaml"
+
+if [[ ! -f ${RESOURCE_FILE} || ! -w ${RESOURCE_FILE} ]]; then
+  echo "${RESOURCE_FILE} does not exist or is not writable"
+  exit 1
+fi
+
+# Get the index of the last boskos project from the resources file
+LAST_INDEX=$(grep -n "knative-boskos-" ${RESOURCE_FILE} | wc -l)
```

Comment:
It's safer not relying on `wc`:

```suggestion
LAST_INDEX=$(grep "knative-boskos-" ${RESOURCE_FILE} | grep -o "[0-9]\+" | sort -nr | head -1)
```

Reply:
Done.